lib/order.rb

```diff
@@ -0,0 +1,61 @@
+# I would like to be able to select some number of several available dishes
+# I would like to check that the total I have been given matches the sum of the various dishes in my order
+
+require_relative 'menu.rb'
+
+class Order
+
+attr_reader :user_order, :place_order, :quantity
+
+  def initialize
+    @user_order = []
+    @basket = Hash.new([0.0])
+    @menu = Menu.new
+  end
+
+  # def basket
+  #   @basket.dup
+  # end
+  #this works but query best for price?
+
+  def price(item)
+  fail 'Item not in menu!' unless !!@menu_contents.include?(item)
+  @menu_contents[item]
+  end
+
+  def place_order(item, quantity = 1)
+    # fail "please order from menu" if !@menu.public_menu_contents.keys.include?(item)
```

Comment:
Metrics/LineLength: Line is too long. [86/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)